app/components/StationDetailView.vue

```diff
@@ -122,6 +132,23 @@ export default Vue.extend({
         console.log("station-detail", this.stationId);
         void this.completeSetup();
     },
+    created() {
+        this.unwatch = this.$s.watch(
```

Comment:
Is this necessary or can we use a more standard watch: { } https://vuejs.org/v2/guide/computed.html#Watchers